### ISSUE TITLE
Scrolling in Edge (Win) throws Warning in console: SCRIPT5007: Unable to get property 'handleEvent' of undefined or null reference

### ISSUE THREAD

**Issue by DaveCole** [schrodinger/fixed-data-table-2#245]
## Expected Behavior
Scrolling with no JS warnings

## Current Behavior
Scrolls fine but with JS warning:
SCRIPT5007: Unable to get property 'handleEvent' of undefined or null reference
EventListener.js (27,1);

## Possible Solution
```
listen: function listen(target, eventType, callback) {
    if (target.addEventListener && callback) {       // "&& callback" added
      target.addEventListener(eventType, callback, false);
      return {
        remove: function remove() {
          target.removeEventListener(eventType, callback, false);
        }
      };
    } else if (target.attachEvent) {
      target.attachEvent('on' + eventType, callback);
      return {
        remove: function remove() {
          target.detachEvent('on' + eventType, callback);
        }
      };
    }
  },
```

## Steps to Reproduce (for bugs)
Scroll any example in IE/Edge on Windows

## Context
I wonder if this contributes to the poor scrolling performance in IE/Edge

## Your Environment

* Version used: Latest
* Browser Name and version: Edge/IE11
* Operating System and version (desktop or mobile): Windows
* Link to your project:

**Comment by wcjordan** [comment]
I'm unable to reproduce, but happy to accept a PR as you've suggested.
Do you have reproduction steps beyond trying the examples in IE11?

It's curious this hasn't been fixed on fbjs yet:
https://github.com/facebook/fbjs/blob/c69904a511b900266935168223063dd8772dfc40/packages/fbjs/src/stubs/EventListener.js

**Comment by pradeepnschrodinger** [comment]
Closing the issue since it's been a long time and it wasn't reproducable from our side. 
Also this should be fixed with [v1.0.0](https://github.com/schrodinger/fixed-data-table-2/releases/tag/v1.0.0). 

Feel free to reopen or open up a new issue otherwise.
